src/main/java/seedu/address/model/interview/Interview.java

```diff
@@ -21,7 +21,7 @@
     public static final String DISPLAY_FORMAT = "MMM dd yyyy , HH:mm"; //e.g. Sep 21 2022, 09:30
     public static final String EMPTY_TIME = "-";
     public static final String MESSAGE_CONSTRAINTS =
-            "Interview time should follow the format: [year-month-date, hour-minute]. "
+            "Interview time should follow the format: yyyy-MM-dd, HH:mm. "
```

Comment:
I think Yao Yi might have changed this part after she had fixed the bugs for interview date and time